ember-client/shared/src/main/scala/org/http4s/ember/client/internal/ClientHelpers.scala

```diff
@@ -80,6 +82,7 @@ private[client] object ClientHelpers {
       unixSockets.client(address),
       tlsContextOpt,
       false,
+      true,
```

Comment:
Hm, should this respect the setting value?

Reply:
What do you mean? I put `true` here to keep the previous behavior.

Reply:
Right, but this PR is all about making it configurable, so shouldn't we use the configuration here?

Reply:
I followed whatever the endpoint check is doing. It's not forwarded. It's the above `false`

Reply:
Well, not exactly. The default value for the endpoint check is `true`, but here it is set to `false`. So it's decidedly going against the default.

I think these should both respect the user configuration. It's principle of least surprise.

Reply:
Ok I forwarded the params. But note that this is change of behavior for the endpoint check

Reply:
Thanks for making the change, and the call-out about the change of behavior. It should be fine, since it's unlikely that TLS over unix sockets is used at all, and if it ever is, I think it's much better for it to be consistent.